lib/node_modules/@stdlib/utils/every-own-by/docs/types/test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import everyOwnBy = require( './index' );
+
+const isPositive = ( v: number ): boolean => {
+	return ( v > 0 );
+};
+
+// TESTS //
+
+var obj = {
+	'a':1,
+	'b':2,
+	'c':3
+};
+// The function returns a boolean...
```

Comment:
Empty line before comment:

```suggestion
};

// The function returns a boolean...
```